.travis.yml

```diff
@@ -17,18 +17,17 @@ env:
   matrix:
     - PLONE_VERSION=4.3
     - PLONE_VERSION=4.3 DEXTERITY_ONLY=true
-    - PLONE_VERSION=4.3 JQUERY=1.9.1.3
+    - PLONE_VERSION=4.3 VERSIONS=versions:plone.app.jquery=1.9.2
```

Comment:
Indeed, it's in official docs, first paragraph :laughing: : http://docs.buildout.org/en/latest/reference.html